keys.tf

```diff
@@ -35,13 +35,13 @@ resource "null_resource" "write_kubeconfig" {
       ssh-keygen -R ${local.external_ip} >/dev/null 2>&1
       until rsync -e "ssh -o ConnectTimeout=5 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" --rsync-path="sudo rsync" ${var.servers[0].system_user}@${local.external_ip}:/etc/rancher/rke2/rke2.yaml rke2.yaml >/dev/null 2>&1; do echo Wait rke2.yaml generation && sleep 5; done \
       && chmod go-r rke2.yaml \
```

Comment:
Can you a way to detect the correct binary, e.g.: `YQ_BIN=(which yq | "python -m yq")` or similar?